packages/css/src/button.css

```diff
@@ -44,27 +44,7 @@
     --dsc-button-color--hover: var(--ds-color-text-default);
   }
   &[data-variant='secondary'] {
-    --dsc-button-border-color: var(--ds-color-border-strong);
-  }
-
-  &[data-color] {
-    --dsc-button-background--active: var(--ds-color-base-active);
-    --dsc-button-background--hover: var(--ds-color-base-hover);
-    --dsc-button-background: var(--ds-color-base-default);
-    --dsc-button-color: var(--ds-color-base-contrast-default);
-    --dsc-button-color--hover: var(--ds-color-base-contrast-default);
-
-    &[data-variant='secondary'],
-    &[data-variant='tertiary'] {
-      --dsc-button-background: transparent;
-      --dsc-button-background--active: var(--ds-color-surface-active);
-      --dsc-button-background--hover: var(--ds-color-surface-hover);
-      --dsc-button-color: var(--ds-color-text-subtle);
-      --dsc-button-color--hover: var(--ds-color-text-default);
-    }
-    &[data-variant='secondary'] {
-      --dsc-button-border-color: var(--ds-color-border-strong);
-    }
+    --dsc-button-border-color: currentColor;
```

Comment:
Comment as to why? (keep same color as text color)

Reply:
change to text-subtle as border should hang together with text color.